### PR TITLE
Securonix Release 203

### PR DESCRIPTION
**Pull request by scnx-jelangovan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added support for handling invalid token error. If an invalid token error occurs, this will reset the integration cache and regenerate the new token on the subsequent execution.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Development container for Content. Poetry hash 022d65df3402ad71ca04cca0b76b759f0874f425330e511e952f4d2ebf9f265f
+// Development container for Content. Poetry hash 2b032773831d4bd21843b6af6885472a9ec80451cd99e2726df6d006f3f62d49
```
Comment (GuyAfik) [inline .devcontainer/devcontainer.json:1]:
```suggestion
// Development container for Content. Poetry hash 022d65df3402ad71ca04cca0b76b759f0874f425330e511e952f4d2ebf9f265f
```

not your fault, but should be reverted.

**Review by GuyAfik — COMMENTED** [review]
@scnx-jelangovan looks great, very nice unit-tests, well done!

gave very small comments, once its solved we can merge.

**Comment by GuyAfik** [comment]
@scnx-jelangovan can you update docker images please? Validations fail here: https://app.circleci.com/pipelines/github/demisto/content/256285/workflows/81b239fb-d0a0-402a-996c-b9b34e3b5d47/jobs/698448

other than that looks incredible!

File: Packs/Securonix/ReleaseNotes/2_0_3.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+##### SecuronixCloseHistoricalXSOARIncidents
+
+- Added a unit test case file for the script.
+- Updated the Docker image to: *demisto/python3:3.10.11.56082*.
+
+##### UpdateSecuronixIncidentStatus
+
+- Added a unit test case file for the script.
+- Updated the Docker image to: *demisto/python3:3.10.11.56082*.
```
[the remark this answers is not in the document]
Comment (scnx-jelangovan) [reply Packs/Securonix/ReleaseNotes/2_0_3.md]: Lines specifying the addition of unit test cases are removed from release notes.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (scnx-jelangovan) [reply .devcontainer/devcontainer.json]: Reverted the file.

**Comment by scnx-jelangovan** [comment]
> @scnx-jelangovan can you update docker images please? Validations fail here: https://app.circleci.com/pipelines/github/demisto/content/256285/workflows/81b239fb-d0a0-402a-996c-b9b34e3b5d47/jobs/698448
> 
> other than that looks incredible!

Thanks, @GuyAfik. 
Updated docker image tag.

**Review by GuyAfik — APPROVED** [review]
@scnx-jelangovan Wonderful work! Thanks for all the work!

**Pull request merged**